homeassistant/components/camera/arlo.py

```diff
@@ -14,20 +14,43 @@
 from homeassistant.components.arlo import DEFAULT_BRAND, DATA_ARLO
 from homeassistant.components.camera import Camera, PLATFORM_SCHEMA
 from homeassistant.components.ffmpeg import DATA_FFMPEG
+from homeassistant.const import ATTR_BATTERY_LEVEL
 
 DEPENDENCIES = ['arlo', 'ffmpeg']
-
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_BRIGHTNESS = 'brightness'
```

Comment:
Is this not in const.py?

Reply:
It isn’t: `CONF_BRIGHTNESS` exists, but `ATTR_BRIGHTNESS` doesn’t.